internal/pkg/utils/etcdhelper/logger.go

```diff
@@ -1,9 +1,11 @@
 package etcdhelper
```

Comment:
Myslim, ze to nema zmysel nejak extra kontrolovat, ... v pripade nejakej chyby/vylepsenia si to opravi ten, kto prvy na to narazi.

Reply:
Da sa to vyskusat s lubovolnym testom, pomocou `TEST_VERBOSE=true`.